### ISSUE TITLE
renaming of the early infantile epileptic encephalopathy series

### ISSUE THREAD

**Issue by sbello** [DiseaseOntology/HumanDiseaseOntology#911]
OMIM is in the process of renaming the early infantile epileptic encephalopathy series to developmental and epileptic encephalopathy.

Looking at https://pubmed.ncbi.nlm.nih.gov/31926847/ it seems likely that we need to review the organization of these terms in the DO. Currently the EIEE series is under neonatal period electroclinical syndrome (DOID:0050702) but DEE covers the span from neonatal to adult. 
The reference provided has a definition of DEE in two parts: 
"An epileptic encephalopathy is defined by the presence of frequent epileptiform activity that impacts adversely on development, typically causing slowing or regression of developmental skills, and usually associated with frequent seizures. Many of the epileptic encephalopathies are now known to have an identifiable molecular genetic basis. The term‘developmental’ was introduced as there are multiple facets leading to developmental impairment in affected individuals."

additional reference https://pubmed.ncbi.nlm.nih.gov/29356691/ has the definition: 
"This differs from the newly introduced concept of 'developmental encephalopathy' which applies to where the epilepsy and developmental delay are both because of the underlying aetiology and aggressive antiepileptic treatment may not be helpful. " Although it is unclear if they just left out "and epileptic" or if they are defining something different. It seems likely to me that they shortened the name and are talking about the same disease class.

**Comment by sbello** [comment]
I think OMIM is done with the renaming, circling back to this now.

**Comment by sbello** [comment]
https://onlinelibrary-wiley-com/doi/10.1111/epi.13709

**Comment by sbello** [comment]
suggest creating a new term:
developmental and epileptic encephalopathy
child of electroclinical syndrome DOID:0050701
definition: An electroclinical syndrome characterized by epileptiform activity and at least one other pathology that together contribute to  cognitive and behavioral impairments including developmental delay or regression with onset anywhere from birth to adulthood.

Most of the terms currently under early infantile epileptic encephalopathy (DOID:0050709) would move out to under this term.

We may want to obsolete early infantile epileptic encephalopathy (DOID:0050709) once this is done.

Note OMIM states "Developmental and epileptic encephalopathies (DEEs) represent a clinically and genetically heterogeneous group of age-dependent neurologic disorders characterized by onset of refractory seizures in infancy or early childhood. "
But ILAE has any age and Fig 2 in PMID: 31926847 has DEE spanning from birth to adult. I went with the broader age span based off of the latter references.

@lschriml if this sounds okay to you I'll start working on renaming these and moving terms this week. I suspect this will take a while as it looks like many of the individual terms have had additional updates. I'll also add age of onset information for the various entries as I skipped this initially since the parent defined age of onset.
